contributors/guide/README.md

```diff
@@ -0,0 +1,212 @@
+**OWNER:**
+
+SIG-ContribX
+
+
+# WARNING
+Hello! This is the starting point for our brand new contributor guide, currently underway as per [-issue#6102](https://github.com/kubernetes/website/issues/6102) and in need of help. Please be patient, or fix a section below that needs improvement, and submit a pull request!
+
+Many of the links below should lead to relevant documents scattered across the community repository. Often, the linked instructions need to be updated or cleaned up. 
+
+* If you do so, please port the relevant file from its previous location to come live in the community/contributors/guide folder, and delete its previous location.
```

Comment:
suggestion: s/port/{move|relocate}/ -- easier to understand, especially with newer english speakers